superset-frontend/src/components/Modal.tsx

```diff
@@ -20,7 +20,7 @@ import React from 'react';
 import styled from '@superset-ui/style';
 import { Modal as BaseModal } from 'react-bootstrap';
 import { t } from '@superset-ui/translation';
-import Button from 'src/views/CRUD/dataset/Button';
+import Button from 'src/views/CRUD/data/dataset/Button';
```

Comment:
FWIW, I'm planning to open a PR soon sweeping through and cleaning out any one-off buttons, including this fella. One button to rule them all! ;) I know this is the newest-looking button, so I'll be supporting various bits and pieces from it.

Reply:
Yes, the new buttons are this one and the really wide CTA one. Maybe they can just be variants of the `bsSize` attribute

Reply:
right... maybe a "cta" prop... to apply these styles would do just fine. It just needs a good name, really :D Anyway, we can discuss this outside of this PR.